dwave/system/testing.py

```diff
@@ -208,16 +325,51 @@ def __init__(self, broken_nodes=None, broken_edges=None,
             'num_reads_range': [1, 10000],
             'per_qubit_coupling_range': [-18.0, 15.0],
             'problem_run_duration_range': [0.0, 10000000.0],
+            'problem_timing_data': {'version': '1.0.0',
+                                    'typical_programming_time': 14072.88,
+                                    'reverse_annealing_with_reinit_prog_time_delta': 0.0,
+                                    'reverse_annealing_without_reinit_prog_time_delta': 5.55,
+                                    'default_programming_thermalization': 1000.0,
+                                    'default_annealing_time': 20.0,
+                                    'readout_time_model': 'pwl_log_log',
+                                    'readout_time_model_parameters': [0.0,
+                                                                      0.7699665876947938,
+                                                                      1.7242758696010096,
+                                                                      2.711975459489206,
+                                                                      3.1639057672764026,
+                                                                      3.750276915153992,
+                                                                      1.539131714800995,
+                                                                      1.8726623164229292,
+                                                                      2.125631787097315,
+                                                                      2.332672340068556,
+                                                                      2.371606651233025,
+                                                                      2.3716219271760215],
+                                    'qpu_delay_time_per_sample': 20.54,
+                                    'reverse_annealing_with_reinit_delay_time_delta': -4.5,
+                                    'reverse_annealing_without_reinit_delay_time_delta': -1.5,
+                                    'default_readout_thermalization': 0.0,
+                                    'decorrelation_max_nominal_anneal_time': 2000.0,
+                                    'decorrelation_time_range': [500.0, 10000.0]},
```

Comment:
If you don't think adding `problem_timing_data` is critical at this point, I would prefer to just use [`qpu_problem_timing_data` mock from the cloud-client](https://github.com/dwavesystems/dwave-cloud-client/pull/530/files#r942863661) (to be added when https://github.com/dwavesystems/dwave-cloud-client/pull/530 is merged).

Until then maybe just say:
```suggestion
            # TODO: add `problem_timing_data` using `dwave.cloud.testing.mocks.qpu_problem_timing_data`
            # (when the mock is merged and released)
```

But we can replace that in a follow-up PR, so we can leave it in for now. (As I noted in my initial review, long-term, I'd like we use low-level solver mocks from the cloud-client for this sampler mock.)

Reply:
Easier to leave for now.

Reply:
Note that your tests shouldn't fail if you set `problem_timing_data=None`, as above (you're testing only `properties.keys()` matches a live solver).